File: frontend/client/lib/models/language.dart
class Language {
  final String languageCode;
  final String languageName;

  Language({
    required this.languageCode,
    required this.languageName,
  });

  factory Language.fromJson(Map<String, dynamic> json) {
    return Language(
      languageCode: json['languageCode'],
      languageName: json['languageName'],
    );
  }
}

File: backend/Server/Models/Language.cs
using Server.Models;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Server.Models
{
    [Table("Languages")]
    public class Language
    {
        [Key]
        [MaxLength(10)]
        public string LanguageCode { get; set; }

        [Required]
        [MaxLength(100)]
        public string LanguageName { get; set; } = string.Empty;

        // Navigation
        public virtual ICollection<Narration> Narrations { get; set; } = new List<Narration>();
        public virtual ICollection<FoodTranslation> FoodTranslations { get; set; } = new List<FoodTranslation>();
        public virtual ICollection<UITranslation> UITranslations { get; set; } = new List<UITranslation>();
    }
}